# Coleta de dados para o exercício de casa:

Coletar a altura média de cada uma das alunas, excluindo a Professora, Facilitaroa e Monitora.


**Exercício: Teste z para Altura Média da Turma**

**Cenário:**

Queremos avaliar se a altura média da turma on35 é significativamente diferente da altura média nacional da população feminina brasileira, que é de 161 cm.

**Objetivo:**

    Verificar se a altura média da turma é significativamente diferente de 160 cm.

Dados da Amostra:

    Altura da turma (em cm): [158, 162, 161, 159, 164, 165, 157, 160, 163, 161, 159, 162, 158, 160, 161]


**Perguntas:**


Calcule a estatística z e o valor p para verificar se a altura média da turma é significativamente diferente de 161 cm.

**Interprete os resultados:** A altura média da turma é significativamente diferente de 161 cm?

**Hipóteses:**

**Hipótese Nula (H0):**

A hipótese nula é a suposição inicial de que não há diferença significativa entre a média da amostra e o valor de referência.
No contexto do nosso exemplo:

    H0​: A altura média da turma on 35 é igual à altura média nacional.
    H0:μ=161

**Hipótese Alternativa (H1):**

A hipótese alternativa é a afirmação que desejamos testar, que sugere que existe uma diferença significativa.

No contexto do nosso exemplo:

      H1​: A altura média dos alunos da turma é diferente da altura média nacional.
      H1:μ≠161

### Em outras palavras...

**Hipótese Nula (H0):**

Supõe que não há diferença entre a média da amostra e a média da população (161 cm). A média da altura das alunas da turma é 160 cm.

**Hipótese Alternativa (H1):**

Sugere que há uma diferença entre a média da amostra e a média da população. A média da altura das alunas da turma não é 160 cm.


**Teste Bilateral:**

Neste exemplo, utilizamos um teste bilateral (ou de duas caudas), pois estamos verificando se a média da altura da turma é diferente (pode ser maior ou menor) da média nacional, e não apenas se é maior ou menor.


In [1]:
import numpy as np
from statsmodels.stats.weightstats import ztest

# Dados da amostra
altura_turma = np.array([158, 162, 161, 159, 164, 165, 157, 160, 163, 161, 159, 162, 158, 160, 161])
media_nacional = 161

# Teste z para uma amostra
z_statistic, p_value = ztest(altura_turma, value=media_nacional)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}, p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: 1.1282, p-value: 0.2593


**Respostas Esperadas:**

Teste z para uma amostra:


    * Se o valor p for menor que 0.05, a altura média da turma é significativamente diferente de 160 cm.

    A interpretação do valor do teste z para uma amostra, com uma estatística z de 1.1282 e um valor p de 0.2593, é a seguinte:

**Interpretação:**

**Estatística z (z-statistic): 1.1282**

  A estatística z indica quantos desvios padrão a média da amostra está distante da média de referência (160 cm) sob a hipótese nula.
  Um valor positivo da estatística z sugere que a média da amostra está acima da média de referência, mas precisamos do valor p para determinar a significância.

**Valor p (p-value): 0.2593**

  O valor p representa a probabilidade de observar um valor da estatística z tão extremo ou mais extremo do que o valor observado, assumindo que a hipótese nula é verdadeira.
  Neste caso, um valor p de 0.2593 significa que há aproximadamente 25.93% de chance de obter a média da amostra observada (ou uma mais extrema) se a média real da população fosse de fato 160 cm.

**Conclusão:**

Nível de Significância: Um nível de significância comum usado em testes estatísticos é 0.05 (ou 5%). Se o valor p for menor que 0.05, rejeitamos a hipótese nula.
Neste caso: O valor p (0.2593) é maior que 0.05. Portanto, não rejeitamos a hipótese nula.

**Interpretação Final:**

A altura média da amostra de alturas da turma não é significativamente diferente de 160 cm. Em outras palavras, não há evidências estatisticamente significativas para afirmar que a altura média da turma difere da altura média nacional de 160 cm.